torchmultimodal/modules/encoders/albef_multimodal_encoder.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+from torch import nn, Tensor
+from torchmultimodal.modules.encoders.albef_text_encoder import (
+    ALBEFTransformerAttention,
+)
+from torchmultimodal.utils.common import get_extended_attention_mask
+
+
+class ALBEFMultimodalEncoder(nn.Module):
+    """
+    Construct multimodal embeddings from image embeddings, text embeddings, and text attention mask.
```

Comment:
Maybe add a bit more detail here explaining how it compares to the text encoder and the use of cross-attention